packages/react/src/users/hooks/__tests__/useUserCloset.test.ts

```diff
@@ -0,0 +1,497 @@
+import { cleanup, renderHook } from '@testing-library/react';
```

Comment:
This file should be called `useUserClosetItems.test.ts`

Reply:
Fixed 👍